src/eventHandlers/TransferEventHandler.ts

```diff
@@ -109,6 +118,9 @@ export default class TransferEventHandler extends EventHandlerBase<'Transfer(add
       dripList.previousOwnerAddress = from;
       dripList.ownerAccountId = await calcAccountId(to);
 
+      // This is real transfer. The Drip List should not be visible unless the block number is less than the visibility threshold.
+      dripList.isVisible = blockNumber < visibilityThresholdBlockNumber;
```

Comment:
doesn't minting a new NFT account produce `transfer` where from === null address? should this set `isVisible` to true if `from` is null address?

Reply:
The _creation_ of the drip list (`from === ZeroAddress`) is covered before in the code.
If the code reaches this point, the list already exists. Makes sense?